homeassistant/components/zwave/__init__.py

```diff
@@ -84,6 +84,16 @@
     vol.Optional(const.ATTR_CONFIG_SIZE, default=2): vol.Coerce(int)
 })
 
+SET_INDICATOR_SCHEMA = vol.Schema({
+    vol.Required(const.ATTR_NODE_ID): vol.Coerce(int),
+    vol.Required(const.ATTR_CONFIG_VALUE): vol.Coerce(int)
+})
+
+REFRESH_NODE_VALUE_SCHEMA = vol.Schema({
+    vol.Required(const.ATTR_NODE_ID): vol.Coerce(int),
+    vol.Required(const.ATTR_VALUE_ID): vol.Any(vol.Coerce(int), cv.string)
```

Comment:
Any reason this can't be just `vol.Coerce(int)` like in other places?

Reply:
Coerce should turn a string into an int. 
It should work - please investigate further.